CMakeLists.txt

```diff
@@ -952,6 +960,11 @@ if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
 		DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/48x48/apps)
 	install(FILES Installer/dolphin-emu.desktop
 		DESTINATION ${CMAKE_INSTALL_PREFIX}/share/applications)
+	# Install manpages
+	install(FILES Installer/dolphin-emu.6
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/usr/share/man/man6)
```

Comment:
You have an extra /usr there -- CMAKE_INSTALL_PREFIX will usually already be /usr.